src/Lykke.Service.Decred.Api/Startup.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Data;
+using System.Net;
+using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using AzureStorage.Tables;
+using Common.Log;
+using DcrdClient;
+using Decred.BlockExplorer;
+using Lykke.Common.ApiLibrary.Middleware;
+using Lykke.Logs;
+using Lykke.Service.Decred.Api.Common;
+using Lykke.Service.Decred.Api.Common.Entity;
+using Lykke.Service.Decred.Api.Common.Services;
+using Lykke.Service.Decred.Api.Middleware;
+using Lykke.Service.Decred.Api.Repository;
+using Lykke.Service.Decred.Api.Services;
+using Lykke.SettingsReader;
+using Lykke.SlackNotification.AzureQueue;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Http;
+using NDecred.Common;
+using Newtonsoft.Json.Serialization;
+using Npgsql;
+
+namespace Lykke.Service.Decred.Api
+{
+    public class Startup
```

Comment:
Use Lykke.Sdk, as it done in sign service

Reply:
I've explained my case on Telegram, this introduces an issue that needs to be addressed by Lykke.  I will not work on this.